src/ucp/core/ucp_context.c

```diff
@@ -81,16 +81,19 @@ static ucs_config_field_t ucp_config_table[] = {
 
   {"TLS", UCP_RSC_CONFIG_ALL,
    "Comma-separated list of transports to use. The order is not meaningful.\n"
-   " - all    : use all the available transports.\n"
-   " - sm/shm : all shared memory transports.\n"
-   " - mm     : shared memory transports - only memory mappers.\n"
-   " - ugni   : ugni_rdma and ugni_udt.\n"
-   " - ib     : all infiniband transports.\n"
-   " - rc     : rc verbs (uses ud for bootstrap).\n"
-   " - rc_x   : rc with accelerated verbs (uses ud_x for bootstrap).\n"
-   " - ud     : ud verbs.\n"
-   " - ud_x   : ud with accelerated verbs.\n"
-   " - dc_x   : dc with accelerated verbs.\n"
+   " - all     : use all the available transports.\n"
+   " - sm/shm  : all shared memory transports.\n"
+   " - mm      : shared memory transports - only memory mappers.\n"
+   " - ugni    : ugni_rdma and ugni_udt.\n"
+   " - ib      : all infiniband transports.\n"
+   " - rc_v    : rc verbs (uses ud for bootstrap).\n"
+   " - rc_x    : rc with accelerated verbs (uses ud_x for bootstrap).\n"
+   " - rc      : rc_v and rc_x (preferably if available) together"
+   " - ud_v    : ud verbs.\n"
+   " - ud_x    : ud with accelerated verbs.\n"
+   " - ud      : ud_v and ud_x (preferably if available) together"
```

Comment:
`.` in the end and I'd remove `together`

Reply:
done